docs/installation/installing-with-npm.md

```diff
@@ -200,6 +198,12 @@ If you're using a bundler such as Webpack, use the `import` syntax to import a c
 import { Radios } from 'govuk-frontend'
 ```
 
+or
+
+```JS
+import Button from 'govuk-frontend/govuk/components/button/button'
```

Comment:
What does this offer over `import { Button } from 'govuk-frontend'`? Are there scenarios when `import { Button } from 'govuk-frontend'` wouldn't work and the longer form is needed?

Just wondering if we should keep this simpler – by providing two options we're giving users another decision to make. If we need both, it'd be good to be clear when you might use each option.

Reply:
Agreed, let's not have two ways here, and keep the original.

Reply:
👍 done